src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -75,7 +75,8 @@
 #define PX_PER_CM 37.79528f //1 in = 2.54 cm -> PX_PER_IN/2.54
 
 
-typedef SvgNode* (*FactoryMethod)(SvgLoaderData* loader, SvgNode* parent, const char* buf, unsigned bufLength);
+typedef bool (*parseAttributes)(const char*, unsigned, simpleXMLAttributeCb, const void*);
```

Comment:
I think you can add parameter names here in order to clarify the function specification.

Reply:
done